css/parser/tests/fixture/important/basic/output.json

```diff
@@ -0,0 +1,676 @@
+            }
+          ]
+        }
+      ],
+      "block": {
+        "type": "DeclBlock",
+        "span": {
+          "start": 3,
+          "end": 495,
+          "ctxt": 0
+        },
+        "properties": [
+          {
+            "type": "Property",
+            "span": {
+              "start": 9,
+              "end": 24,
+              "ctxt": 0
+            },
+            "name": {
+              "type": "Text",
+              "span": {
+                "start": 9,
+                "end": 13,
+                "ctxt": 0
+              },
+              "value": "prop"
+            },
+            "values": [
+              {
+                "type": "Text",
+                "span": {
+                  "start": 15,
+                  "end": 24,
+                  "ctxt": 0
+                },
+                "value": "important"
+              }
+            ],
+            "important": null
+          },
+          {
+            "type": "Property",
+            "span": {
+              "start": 30,
+              "end": 50,
+              "ctxt": 0
+            },
+            "name": {
+              "type": "Text",
+              "span": {
+                "start": 30,
+                "end": 35,
+                "ctxt": 0
+              },
+              "value": "color"
+            },
+            "values": [
+              {
+                "type": "Text",
+                "span": {
+                  "start": 37,
+                  "end": 40,
+                  "ctxt": 0
+                },
+                "value": "red"
+              },
+              {
+                "type": "Text",
+                "span": {
+                  "start": 41,
+                  "end": 50,
+                  "ctxt": 0
+                },
+                "value": "important"
+              }
+            ],
+            "important": null
+          },
+          {
+            "type": "Property",
+            "span": {
+              "start": 56,
+              "end": 76,
+              "ctxt": 0
+            },
+            "name": {
+              "type": "Text",
+              "span": {
+                "start": 56,
+                "end": 61,
+                "ctxt": 0
+              },
+              "value": "width"
+            },
+            "values": [
+              {
+                "type": "UnitValue",
+                "span": {
+                  "start": 63,
+                  "end": 66,
+                  "ctxt": 0
+                },
+                "value": {
+                  "type": "Number",
+                  "span": {
+                    "start": 63,
+                    "end": 64,
+                    "ctxt": 0
+                  },
+                  "value": 1.0
+                },
+                "unit": {
+                  "span": {
+                    "start": 64,
+                    "end": 66,
+                    "ctxt": 0
+                  },
+                  "kind": null
```

Comment:
Why we don't write kind of unit in tests?

Reply:
I think it's a bug.

Reply:
:+1: